### ISSUE TITLE
Embed a Jupyter notebook in the viewer

### ISSUE THREAD

**Issue by yungyuc** [solvcon/solvcon#127]
I have not known how to make it happen, but it would be convenient if the viewer has a Jupyter notebook in it.

This feature will depend on or relate to #16 and #17.

**Comment by ThreeMonth03** [comment]
I just found a project that called [qtconsole](https://github.com/spyder-ide/qtconsole). Is it helpful for this issue?

<img width="594" height="1038" alt="Image" src="https://github.com/user-attachments/assets/803eca0b-9171-42d8-bf3b-269410465500" />

**Comment by yungyuc** [comment]
Yes. The idea of embedding the Jupyter notebook into the viewer (now named "pilot") was inspired by `qtconsole`.
